tools/metrics/histograms/metadata/cobalt/enums.xml

```diff
@@ -37,6 +37,13 @@ https://chromium.googlesource.com/chromium/src.git/+/HEAD/tools/metrics/histogra
   <int value="1" label="True"/>
 </enum>
 
+
```

Comment:
Nit: kill a new line before and ever this enum stanza

Reply:
done